force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -110,7 +190,29 @@ public with sharing class FlowControlLoopsCollections {
      * @return The grade for the given score.
      */
     public static String determineGrade(Integer score) {
-        return null; // Replace null with the variable you used to store the result
+
+         if (score >= 90) {
+            System.debug('A');
+            return 'A';
+
+        }if (score >= 80) {
+            System.debug('B');
+            return 'B';
+
+        }if (score >= 70) {
+            System.debug('C');
+            return 'C';
+
+        }if (score >= 60) {
+            System.debug('D');
+            return 'D';
+
+        }else {
+            System.debug('F');
+            return 'F';
```

Comment:
Watch out for excessive system.debug() usage - you should use it when you as the developer want to see the output of the code at a certain point.

Reply:
Thank you. I will take note.